src/CryptSync.cpp

```diff
@@ -166,7 +166,7 @@ int APIENTRY _tWinMain(HINSTANCE hInstance,
         bool         use7Z        = !!parser.HasKey(L"use7z");
         bool         useGpg       = !!parser.HasKey(L"useGPG");
         bool         fat          = !!parser.HasKey(L"fat");
-        bool         syncDeleted  = !!parser.HasKey(L"syncdeleted");
+        bool         syncDeletions  = !parser.HasKey(L"nosyncdeleted");
         int          compresssize = _wtoi(parser.GetVal(L"compresssize"));
```

Comment:
When testing command line, Windows was throwing an exception here: Unhandled exception at 0x00ED4916 in CryptSync.exe: An invalid parameter was passed to a function that considers invalid parameters fatal.

I had to supply this parameter to complete test